### PR TITLE
Export/vp 250 replace get from type with reflection utility

### PR DESCRIPTION
**Pull request by akak1977**
Also, resolved minor CultureInfo-related issues in csv export tests.

### CODE REVIEW

File: Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/CsvExportProviderTests.cs [hunk 1]
```diff
@@ -383,7 +384,7 @@ public Task ExportPricelists_UseCustomMap()
                 }
             };
 
-            var configuration = new Configuration();
+            var configuration = new Configuration(cultureInfo: CultureInfo.GetCultureInfo("en-US"));
```
Comment (yecli) [inline Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/CsvExportProviderTests.cs]: Could we use InvariantCulture?

File: Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/CsvExportProviderTests.cs [hunk 2]
```diff
@@ -459,7 +460,7 @@ public Task ExportPricelists_SameTypePropertiesMapping()
 
         private string SerializeAndRead(ExportedTypeMetadata metadata, IEnumerable items, Configuration configuration = null)
         {
-            var csvConfiguration = new CsvProviderConfiguration() { Configuration = configuration ?? new Configuration()};
+            var csvConfiguration = new CsvProviderConfiguration() { Configuration = configuration ?? new Configuration(cultureInfo: CultureInfo.GetCultureInfo("en-US")) };
```
Comment (yecli) [inline Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/CsvExportProviderTests.cs]: Could we use InvariantCulture?

File: Modules/vc-module-export/VirtoCommerce.ExportModule.Data/Extensions/ExportedTypeMetadataExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            public PropertyInfo PropertyInfo { get; set; }
+        }
+
+        /// <summary>
+        /// Returns basic info about potentially exported fields of entity.
+        /// </summary>
+        /// <typeparam name="T">Type for getting metadata.</typeparam>
+        /// <param name="extractReferenceProperties">If true, reference properties properties (<see cref="Enitity"/> and collection of <see cref="Enitity"/>) will be extracted recursively.</param>
+        /// <returns>Metadata for the T type.</returns>
+        public static ExportedTypeMetadata GetFromType(this Type type, bool extractReferenceProperties)
```
Comment (yecli) [inline Modules/vc-module-export/VirtoCommerce.ExportModule.Data/Extensions/ExportedTypeMetadataExtensions.cs]: Let's rename to `GetPropertyNames`, as we use this as extension

Comment (akak1977) [reply Modules/vc-module-export/VirtoCommerce.ExportModule.Data/Extensions/ExportedTypeMetadataExtensions.cs]: OK

File: Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/CsvExportProviderTests.cs (hunk 1 above)
Comment (akak1977) [reply Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/CsvExportProviderTests.cs]: OK

File: Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/CsvExportProviderTests.cs (hunk 2 above)
Comment (akak1977) [reply Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/CsvExportProviderTests.cs]: OK

**Review by yecli — APPROVED** [review]

**Pull request merged**